events.go

```diff
@@ -227,7 +227,13 @@ type DOMTransactionEvent struct{ *BasicEvent }
 type DragEvent struct{ *BasicEvent }
 type EditingBeforeInputEvent struct{ *BasicEvent }
 type ErrorEvent struct{ *BasicEvent }
+
```

Comment:
Why the newline here?

Reply:
For consistency.

See [here](https://github.com/dominikh/go-js-dom/blob/6a8015da13438f4de2c715e781271ab856b4b24c/events.go#L211-L218) and [here](https://github.com/dominikh/go-js-dom/blob/6a8015da13438f4de2c715e781271ab856b4b24c/events.go#L268-L291) for example.

Reply:
Your examples aren't relevant, but sure, let's add a newline.

Reply:
I see now that I misinterpreted the pattern, and those are indeed not necessary relevant. But I’ll  keep the newline since it’s not breaking any precedent, reads a little better (IMO), and you’re ok with it.